src/integration/pages/articles/portuguese/canonical.test.js

```diff
@@ -4,9 +4,12 @@
  */
 
 import { runFlourishCanonicalEmbedTests } from '../../../common';
+import runInlineLinkTests from '../inlineLink';
 
 describe('Canonical', () => {
   describe(pageType, () => {
     runFlourishCanonicalEmbedTests();
+    runInlineLinkTests();
+    // WE NEED A LTR INLINE LINK TEST BUT NEED TO MAKE FIXTURE DATA FOR THIS
```

Comment:
Can you find an article in an LTR service with an inlineLink, and add it?
```suggestion
    runInlineLinkTests();
```

Reply:
I have added them into a persian test, but for some reasons lots of the tests are skipped for it and I don't know why... 🤔

Reply:
Forgot to comment here. It was because I had left in a .skip 🤦‍♀️